frontend/src/components/cluster-dashboard/cluster-list-view/ClusterListView.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
You should look up Enzyme and Jest's snapshot tool to unit test React components. The coverage on this is not great and the tools are super easy to use.

Reply:
Will do, I am gonna start looking into those tools some time next week and start writing unit tests. Meanwhile, if you could add your comments here https://github.com/Headstorm/shukra/issues/10 that would be great!